content/docs/dvclive/get-started.md

```diff
@@ -3,164 +3,189 @@
-project:
+You can now
+[run experiments](/doc/user-guide/experiment-management/running-experiments)
+with `dvc exp run`:
 
 ```dvc
-$ tree
-├── dvclive
-│   ├── images
-│   │   └── img.png
-│   ├── params.yaml
-│   ├── plots
-│   │   └── roc.json
-│   └── scalars
-│       └── acc.tsv
-└── dvclive.json
+$ dvc exp run
 ```
 
-The contents of the `dvclive` folder and `dvclive.json` will vary depending on
-the type of data you have logged and whether you have updated the step value or
-not.
+## Compare and visualize experiments
 
-See `Live.log()`, `Live.log_param()`/`Live.log_params()`, `Live.log_image()` and
-`Live.log_plot()` for more details.
+After following the above steps, you have enabled different ways of monitoring
+the training, comparing, and visualizing the experiments:
 
-### Metrics report
+### DVC
 
-If and when `step` is updated, DVCLive generates or updates a report in
-`dvclive/report.{format}` which will contain all the logged data.
+The [metrics history](/doc/dvclive/api-reference/live/log#step-updates)
+`dvclive/scalars` can be visualized with `dvc plots`.
 
-![](/img/dvclive-html.gif)
+The [metrics summary](/doc/dvclive/api-reference/live/log#description) in
+`dvclive.json` can be visualized with `dvc metrics`.
 
-The `format` can be HTML) or Markdown depending on the value of the `report`
-argument passed to [`Live()`](/doc/dvclive/api-reference/live#parameters).
+If you have logged parameters with `Live.log_param()`, they will be stored in
+`dvclive/params.yaml` and can be visualized with `dvc params`.
 
-<admon type="info">
+In addition, the `dvc exp show` and `dvc exp diff` commands can be used to
+compare both metrics and parameters across experiments.
 
-If you don't update the step number, the report won't be generated unless you
-call `Live.make_report()` directly.
+### Visual Studio Code
 
-</admon>
+If you have installed the
+[DVC Extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc),
+you can monitor the training and compare and visualize experiments using the
+[`Experiments`](https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/experiments-table.md)
+and
+[`Plots`](https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/plots.md)
+views.
+
+![Experiments view](https://github.com/iterative/vscode-dvc/raw/main/extension/resources/walkthrough/images/experiments-table.png)
+![Plost view](https://github.com/iterative/vscode-dvc/raw/main/extension/resources/walkthrough/images/plots-trends.png)
 
-## What next?
+### Iterative Studio
 
-Learn how to use DVCLive alongside other tools:
+After the experiment has finished and you have committed and pushed the results,
+[Iterative Studio](/doc/studio) will automatically parse the outputs generated
+by DVCLive, allowing you to
+[compare and visualize experiments](/doc/studio/user-guide/projects-and-experiments/visualize-and-compare):
```

Comment:
To distinguish from the other products above, it might be better to focus on sharing experiments instead of comparing and visualizing.